src/solc_0.8/ReferralValidator/ReferralValidator08.sol

```diff
@@ -0,0 +1,186 @@
+/* solhint-disable not-rely-on-time, func-order */
+
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.2;
+
+import "../common/Libraries/SigUtil.sol";
+import "../common/Libraries/SafeMathWithRequire.sol";
```

Comment:
I can't find where SigUtil and SafeMathWithRequire are used

Reply:
SafeMathWithRequire is being used and SigUtil was removed -> c21e258